package.json

```diff
@@ -109,7 +109,7 @@
   },
   "fluidBuild": {
     "repoPackages": {
-      "client":  [
+      "client": [
```

Comment:
Similar here, I have no real objection to the formatting changes but it's best to scope out any unintentional changes from your PR.

Reply:
I agree. If you are already touching this file, then it is okay but otherwise, you should avoid such changes.

---
In reply to: [491242429](https://github.com/microsoft/FluidFramework/pull/3577#discussion_r491242429) [](ancestors = 491242429)